### ISSUE TITLE
custom contenttype slugs is broken for listings

### ISSUE THREAD

**Issue by jadwigo** [bolt/bolt#4673]
Suppose you have a simple contenttype like the following:

```yml
custompages:
  name: Pages
  singular_name: Page
  fields:
    title:
      type: text
      required: true
      label: "Title"
    slug:
      type: slug
      uses: title
    body:
      type: textarea
      label: "Body"
```

Then the listing page will be accessible at http://example.com/custompages and thesingle page will be available at http://example.com/page/1 or http://example.com/page/{slug} 

But this breaks down as soon as you want to customize that with your own tablename, multiple and singular slug.

So if you for example add a 'custom'  slug

```yml
custompages:
  name: Pages
  singular_name: Page
  slug: custom
  fields:
    title:
      type: text
      required: true
      label: "Title"
    slug:
      type: slug
      uses: title
    body:
      type: textarea
      label: "Body"
```

Both the http://example.com/custompages and http://example.com/custom are inacessible

(this might be loosely related to https://github.com/bolt/bolt/issues/4623 because those errors also pop up)

**Comment by C4Grey** [comment]
Maybe it is better to use a custom route in that case?

**Comment by jadwigo** [comment]
This is something that worked and is documented in the previous versions of bolt, and it makes sense that it keeps working in the new release too.

So a custom route is not a good solution for this.

**Comment by rossriley** [comment]
I'm pretty sure that this is fixed now by #4841

Testing the above contenttypes both the frontend slug /custom and the backend one seem to work ok now.
